RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -325,11 +325,113 @@ void handlePreAdventure(location place)
 		}
 	}
 
+
+	// ML adjustment zone section
+	boolean doML = true;
+	boolean removeML = false;
+		// removeML MUST be true for purgeML to be used. This is only used for -ML locations like Smut Orc, and you must have 5+ SGEAs to use.
+		boolean purgeML = false;
+
+	boolean[location] highMLZones = $locations[Oil Peak, The Typical Tavern Cellar, The Haunted Boiler Room];
```

Comment:
Isnt this where we added defiled cranny (thats the ml one right). Seems to have been misplaced. Or missed initially or something...

Reply:
Ah yeah, split before that merge. Fixed.